logic/path/src/channel_graph.rs

```diff
@@ -338,51 +452,141 @@ mod tests {
     }
 
     #[test]
-    fn test_channel_graph_self_addr() {
+    fn channel_graph_self_addr() {
```

Comment:
So we are not prefixing with `_test` anymore in general?

Reply:
It's not needed IMHO